sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/jdbc/JdbcUtils.scala

```diff
@@ -672,6 +674,7 @@ object JdbcUtils extends Logging {
           }
           stmt.addBatch()
           rowCount += 1
```

Comment:
We cannot move `rowCount` outside `try` then just use it for the metric?

Reply:
It's used to determine whether it needs one more flush or not at the end of iterating. We'll have one more variable for taking this into account anyway.

Reply:
Ur, I see.

Reply:
Can you leave some comments somewhere about [the policy](https://github.com/apache/spark/pull/26109#discussion_r337864313) to collect metrics?

Reply:
Just added it. 6e908d1

Reply:
Thanks!